cmd/karavictl/cmd/tenant.go

```diff
@@ -30,6 +30,8 @@ import (
 	"github.com/spf13/cobra"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
+	"google.golang.org/protobuf/encoding/protojson"
```

Comment:
Generic comment to look at all files and update copyright.

Reply:
I will raise another PR to update the copyright info and will work with Novus on this without affecting the progress of this one